.gitignore

```diff
@@ -109,3 +109,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# VirtualEnv
+.venv
```

Comment:
Need a trailing slash for a folder.

Reply:
Not sure if we need to specify that `.venv` must be a directory, but adding it anyways for clarity.